examples/index.json

```diff
@@ -871,7 +871,7 @@
         "isLearnByExample": true
       },
       {
-        "name": "Iterating over xml with query",
+        "name": "Iterating over XML with a query",
```

Comment:
Should we use, "Iterate" here to be consistent with other titles?

Reply:
Fixed the above. However, changing the below seems to be confusing:

1. `Querying tables` -> `Query tables`
2. `Querying with streams` -> `Query with streams`

WDYT? @MaryamZi @hasithaa

Reply:
I think they (suggested changes) are OK.

Reply:
Or just "Query streams" for the second one.

Reply:
Updated.